server/src/test/java/org/eclipse/openvsx/search/ElasticSearchServiceTest.java

```diff
@@ -276,6 +284,16 @@ static class MockIndex {
     
     @TestConfiguration
     static class TestConfig {
+        @Bean
+        public MeterRegistry registry() {
+            return new SimpleMeterRegistry();
+        }
+
+        @Bean
+        public UrlConfigService urlConfigService() {
+            return new UrlConfigService();
+        }
+        
```

Comment:
Can't you add these two beans to the `@MockBean` list? These beans are not used in the test.